docs/whats_new/v0.4.1.md

```diff
@@ -1,9 +1,192 @@
+  - Now users can customize model names by using the internal registered names (`timm_image`, `hf_text`, `clip`, `numerical_mlp`, `categorical_mlp`, and `fusion_mlp`) as prefixes. This is helpful when users want to simultaneously use two models of the same type, e.g., `hf_text`. They can just use names `hf_text_0` and `hf_text_1`.
+
+- Support `standalone` feature in `TextPredictor` (#1651) @yiqings
+
+- Fixed saving and loading tokenizers and text processors (#1656) @zhiqiangdon
+  - Saved pre-trained huggingface tokenizers separately from the data processors.
+  - This change is backwards-compatibile with checkpoints saved by verison `0.4.0`.
+
+- Change load from a classmethod to staticmethod to avoid incorrect usage. (#1697) @sxjscience
+
+- Added `AutoMMModelCheckpoint` to avoid evaluating the models to obtain the scores (#1716) @sxjscience
+  - checkpoint will save the best_k_models into a yaml file so that it can be loaded later to determine the path to model checkpoints.
+
+- Extract column features from AutoMM's model outputs (#1718)  @zhiqiangdon
+  - Add one util function to extract column features for both image and text.
+  - Support extracting column features for models `timm_image`, `hf_text`, and `clip`.
+
+- Make AutoMM dataloader return feature column information (#1710) @zhiqiangdon
+
+### Bug fixes
+
+- Fixed calling `save_pretrained_configs` in `AutoMMPrediction.save(standalone=True)` when no fusion model exists ([here](https://github.com/awslabs/autogluon/blob/5a323641072431091d2be5e6dbef5a87b646a408/text/src/autogluon/text/automm/utils.py#L644 )) (#1651) @yiqings
+
+- Fixed error raising for setting key that does not exist in the configuration (#1613) @sxjscience
+
+- Fixed warning message about bf16. (#1625) @sxjscience
+
+- Fixed the corner case of calculating the gradient accumulation step (#1633) @sxjscience
+
+- Fixes for top-k averaging in the multi-gpu setting (#1707) @zhiqiangdon
+
+## Tabular
+
+- Limited RF `max_leaf_nodes` to 15000 (previously uncapped) (#1717) @Innixma
+  - Previously, for very large datasets RF/XT memory and disk usage would quickly become unreasonable. This ensures that at a certain point RF and XT will no longer become larger given more rows of training data. Benchmark results showed that the change is an improvement, particularly for the `high_quality` preset.
+
+- Limit KNN to 32 CPUs to avoid OpenBLAS error (#1722) @Innixma
+  - Issue  #1020. When training K-nearest-neighbors (KNN) models, sometimes a rare error can occur that crashes the entire process:
+
+    ```
+    BLAS : Program is Terminated. Because you tried to allocate too many memory regions.
+    Segmentation fault: 11
+    ```
+
+    It has so far only occurred on machines with >300 GB of memory (2 confirmed instances). In both cases, the system was not low on memory and had plenty to complete the task, yet the error still occurred.
```

Comment:
Can instead say: "This error occurred when the machine had many CPU cores (>64 vCPUs) due to too many threads being created at once. By limiting to 32 cores used, the error is avoided."

Reply:
updated